core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -447,6 +447,7 @@ class AdminManager(val config: KafkaConfig,
       alterConfigOp.opType() match {
         case OpType.SET => Log4jController.logLevel(loggerName, logLevel)
         case OpType.DELETE => Log4jController.unsetLogLevel(loggerName)
+        case _ => // ignore
```

Comment:
The latest version of this file has been updated to explicitly ignore a couple of specific cases.  Maybe we want this to error in the case of an unexpected input?